extensions/redcore.xml

```diff
@@ -57,7 +57,7 @@
         <menu view="dashboard">COM_REDCORE</menu>
         <submenu>
             <menu link="option=com_redcore&amp;view=dashboard" view="dashboard">COM_REDCORE_DASHBOARD</menu>
-            <menu link="option=com_redcore&amp;view=config&amp;layout=edit&amp;component=com_redcore" view="configs">COM_REDCORE_CONFIGURATION</menu>
+            <menu link="option=com_redcore&amp;view=configs" view="configs">COM_REDCORE_CONFIGURATION</menu>
```

Comment:
actually this link should go directly to Config, like it was, why are you changing it? :)